app/uk/gov/hmrc/vatsubscription/models/PendingChanges.scala

```diff
@@ -23,33 +23,40 @@ import uk.gov.hmrc.vatsubscription.models.get.PPOBGet
 
 case class PendingChanges(ppob: Option[PPOBGet],
                           bankDetails: Option[BankDetails],
-                          returnPeriod: Option[ReturnPeriod])
+                          returnPeriod: Option[ReturnPeriod],
+                          mandationStatus: Option[MandationStatus])
 
 object PendingChanges {
 
   private val ppobPath = __ \ "PPOBDetails"
   private val bankDetailsPath =  __ \ "bankDetails"
   private val returnPeriodPath = __ \ "returnPeriod"
+  private val mandationStatusDesPath = __ \ "mandationStatus" \ "mandationStatus"
+  private val mandationStatusWritesPath = __ \ "mandationStatus"
 
   val currentReads: Reads[PendingChanges] = (
```

Comment:
Is there much point in updating this as it's not used. We could just check with the other team and delete it?